src/test/shared/utilities/childProcess.test.ts

```diff
@@ -409,7 +404,7 @@ describe('ChildProcess', async () => {
     function writeBatchFileWithDelays(filename: string): void {
         const file = `
         @echo hi
-        SLEEP 2
+        SLEEP 20
```

Comment:
this makes the effect less prone to accidents on slow systems (e.g. CI). 

The tests are always fast as long as they don't fail. And with this PR, the tests are actually faster because of the use of `waitUntil()` instead of a hardcoded 100ms wait.